frontend/src/App.tsx

```diff
@@ -1,26 +1,24 @@
-import {LoginForm} from './components/loginForm';
-import {SignUp} from './components/signup';
-import { TempLandingPage } from './components/tempLandingPage';
-import './index.css';
-import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
-import { Contacts } from './components/contacts';
-function App() {
+import { LoginForm } from './components/loginForm'
+import { SignUp } from './components/signup'
+import { TempLandingPage } from './components/tempLandingPage'
+import './index.css'
+import { Routes, Route } from 'react-router-dom'
+import { Contacts } from './components/contacts'
+import { PlanForm } from './components/PlanForm'
+import React from 'react'
+
+function App (): JSX.Element {
   return (
-<div> 
-    <Routes> 
-      <Route path = "/login" element = {<LoginForm/>}> </Route>
-    </Routes>
-    <Routes> 
-      <Route path = "/signup" element={<SignUp/>}>  </Route>
-    </Routes>
-    <Routes>
-      <Route path = "/tempLandingPage" element={<TempLandingPage/>}> </Route>
-    </Routes>
-    <Routes>
-      <Route path = "/contactPage" element={<Contacts/>}> </Route>
-    </Routes>
-</div>
-  );
+    <div>
+      <Routes>
+        <Route path="/login" element={<LoginForm />} />
+        <Route path="/signup" element={<SignUp />} />
+        <Route path="/tempLandingPage" element={<TempLandingPage />} />
+        <Route path="/contactPage" element={<Contacts />} />
+        <Route path="/planForm" element={<PlanForm />} />
```

Comment:
not necessary in this pr but think about how you would protect certain pages behind the auth wall. how could we prevent logged out users from accessing logged in pages?

Reply:
I'm currently on this in another branch, we would check if the user is logged in with useContext, if user is not logged in then they won't be able to access certain routes. I would have to consult with you and kevin to implement this correctly.